make/dev/nodeclarity/entrypoint.sh

```diff
@@ -35,10 +35,14 @@ npm link
 cd /harbor_src/ui_ng
 npm link harbor-ui
 
+## Rollup
 ./node_modules/.bin/ngc -p tsconfig-aot.json
 sed -i 's/* as//g' src/app/shared/gauge/gauge.component.js
 ./node_modules/.bin/rollup -c rollup-config.js
 
+## Unlink
+npm unlink harbor-ui
```

Comment:
Add unlink will fix that unauthorized error? I am not challenge this. Just asking.